lib/spack/spack/build_environment.py

```diff
@@ -109,7 +109,7 @@ def __call__(self, *args, **kwargs):
         disable = env_flag(SPACK_NO_PARALLEL_MAKE)
         parallel = not disable and kwargs.get('parallel', self.jobs > 1)
 
-        if parallel:
+        if parallel and 'bazel' not in self.command:
```

Comment:
How about just `BuildCommand` since it doesn't always have to be parallel? Although that's the only thing that separates it from a regular `Executable`. I would be fine with either.